ofl/splinesans/DESCRIPTION.en_us.html

```diff
@@ -0,0 +1,5 @@
+<p>Spline Sans is a Grotesque purpose-built for UI interfaces, checkout processes, and paragraphs of text. Space efficiency is accomplished by condensing traditional grotesque proportions. The cool and restrained tone is accented with strategic "thorn" traps, which blossom into view when set at larger sizes.</p> 
+<p>Spline Sans is an original typeface initiated by the <a href="https://spline.design/">Spline Team</a> and designed by Eben Sorkin and Mirko Velimirovic.
+Project manager - Faride Mereb. Testing and design feedback - Gonçalo Teixeira. Concept - Alejandro León.</p>
+<p>Spline Sans supports Google Core, Google Plus, Google Pro, Plus Optional, and Pro optional glyph sets, enabling the typesetting of English, and Western European languages.</p>
```

Comment:
The names of the google fonts glyphsets are probably going to change in the future and don't make much sense to users

Reply:
That's all good. I think Eben is going to push a small amount of kerning fixes as well in the next week.

Reply:
Yes, I am waiting for the next meeting so we improve the entire build etc ;) happy thanksgiving

Reply:
I fixed the description file